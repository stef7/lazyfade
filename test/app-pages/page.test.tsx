import { render, screen } from "@testing-library/react";
import Home from "@/app-pages/page";

describe("Home", () => {
  it("renders a heading", () => {
    render(<Home />);

    const heading = screen.getByRole("heading", {
      name: /welcome to next\.js!/i,
    });

    expect(heading).toBeInTheDocument();
  });

  it("renders homepage unchanged", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
