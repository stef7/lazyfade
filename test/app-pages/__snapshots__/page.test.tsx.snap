// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders homepage unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <main>
      <h1
        class="title"
      >
        Welcome to 
        <a
          href="https://nextjs.org"
        >
          Next.js!
        </a>
      </h1>
      <p
        class="description"
      >
        Get started by editing 
        <code>
          pages/index.js
        </code>
      </p>
      <div
        class="grid"
      >
        <a
          class="card"
          href="https://nextjs.org/docs"
        >
          <h3>
            Documentation →
          </h3>
          <p>
            Find in-depth information about Next.js features and API.
          </p>
        </a>
        <a
          class="card"
          href="https://nextjs.org/learn"
        >
          <h3>
            Learn →
          </h3>
          <p>
            Learn about Next.js in an interactive course with quizzes!
          </p>
        </a>
        <a
          class="card"
          href="https://github.com/vercel/next.js/tree/canary/examples"
        >
          <h3>
            Examples →
          </h3>
          <p>
            Discover and deploy boilerplate example Next.js projects.
          </p>
        </a>
        <a
          class="card"
          href="https://vercel.com/new"
        >
          <h3>
            Deploy →
          </h3>
          <p>
            Instantly deploy your Next.js site to a public URL with Vercel.
          </p>
        </a>
      </div>
      <ul>
        <li>
          <img
            alt="unsplash 0"
            data-nimg="1"
            decoding="async"
            height="400"
            loading="lazy"
            sizes="400w"
            src="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=3840&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=16&q=75 16w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=32&q=75 32w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=48&q=75 48w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=64&q=75 64w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=96&q=75 96w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=128&q=75 128w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D00.8889161923365612&w=3840&q=75 3840w"
            style="color: transparent; opacity: 0; transition: opacity .5s linear; object-fit: cover;"
            width="400"
          />
        </li>
        <li>
          <img
            alt="unsplash 1"
            data-nimg="1"
            decoding="async"
            height="400"
            loading="lazy"
            sizes="400w"
            src="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=3840&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=16&q=75 16w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=32&q=75 32w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=48&q=75 48w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=64&q=75 64w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=96&q=75 96w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=128&q=75 128w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D10.05974956215706961&w=3840&q=75 3840w"
            style="color: transparent; opacity: 0; transition: opacity .5s linear; object-fit: cover;"
            width="400"
          />
        </li>
        <li>
          <img
            alt="unsplash 2"
            data-nimg="1"
            decoding="async"
            height="400"
            loading="lazy"
            sizes="400w"
            src="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=3840&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=16&q=75 16w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=32&q=75 32w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=48&q=75 48w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=64&q=75 64w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=96&q=75 96w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=128&q=75 128w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D20.8985118916592381&w=3840&q=75 3840w"
            style="color: transparent; opacity: 0; transition: opacity .5s linear; object-fit: cover;"
            width="400"
          />
        </li>
        <li>
          <img
            alt="unsplash 3"
            data-nimg="1"
            decoding="async"
            height="400"
            loading="lazy"
            sizes="400w"
            src="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=3840&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=16&q=75 16w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=32&q=75 32w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=48&q=75 48w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=64&q=75 64w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=96&q=75 96w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=128&q=75 128w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D30.2105628087453686&w=3840&q=75 3840w"
            style="color: transparent; opacity: 0; transition: opacity .5s linear; object-fit: cover;"
            width="400"
          />
        </li>
        <li>
          <img
            alt="unsplash 4"
            data-nimg="1"
            decoding="async"
            height="400"
            loading="lazy"
            sizes="400w"
            src="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=3840&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=16&q=75 16w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=32&q=75 32w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=48&q=75 48w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=64&q=75 64w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=96&q=75 96w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=128&q=75 128w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D40.19879458892645774&w=3840&q=75 3840w"
            style="color: transparent; opacity: 0; transition: opacity .5s linear; object-fit: cover;"
            width="400"
          />
        </li>
        <li>
          <img
            alt="unsplash 5"
            data-nimg="1"
            decoding="async"
            height="400"
            loading="lazy"
            sizes="400w"
            src="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=3840&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=16&q=75 16w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=32&q=75 32w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=48&q=75 48w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=64&q=75 64w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=96&q=75 96w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=128&q=75 128w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D50.9965846501934976&w=3840&q=75 3840w"
            style="color: transparent; opacity: 0; transition: opacity .5s linear; object-fit: cover;"
            width="400"
          />
        </li>
        <li>
          <img
            alt="unsplash 6"
            data-nimg="1"
            decoding="async"
            height="400"
            loading="lazy"
            sizes="400w"
            src="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=3840&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=16&q=75 16w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=32&q=75 32w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=48&q=75 48w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=64&q=75 64w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=96&q=75 96w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=128&q=75 128w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D60.22233020755606248&w=3840&q=75 3840w"
            style="color: transparent; opacity: 0; transition: opacity .5s linear; object-fit: cover;"
            width="400"
          />
        </li>
        <li>
          <img
            alt="unsplash 7"
            data-nimg="1"
            decoding="async"
            height="400"
            loading="lazy"
            sizes="400w"
            src="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=3840&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=16&q=75 16w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=32&q=75 32w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=48&q=75 48w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=64&q=75 64w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=96&q=75 96w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=128&q=75 128w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D70.5413086774530982&w=3840&q=75 3840w"
            style="color: transparent; opacity: 0; transition: opacity .5s linear; object-fit: cover;"
            width="400"
          />
        </li>
        <li>
          <img
            alt="unsplash 8"
            data-nimg="1"
            decoding="async"
            height="400"
            loading="lazy"
            sizes="400w"
            src="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=3840&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=16&q=75 16w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=32&q=75 32w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=48&q=75 48w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=64&q=75 64w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=96&q=75 96w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=128&q=75 128w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D80.1800834668688318&w=3840&q=75 3840w"
            style="color: transparent; opacity: 0; transition: opacity .5s linear; object-fit: cover;"
            width="400"
          />
        </li>
        <li>
          <img
            alt="unsplash 9"
            data-nimg="1"
            decoding="async"
            height="400"
            loading="lazy"
            sizes="400w"
            src="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=3840&q=75"
            srcset="/_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=16&q=75 16w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=32&q=75 32w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=48&q=75 48w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=64&q=75 64w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=96&q=75 96w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=128&q=75 128w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=256&q=75 256w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=384&q=75 384w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fsource.unsplash.com%2Frandom%3Ft%3D90.9080497384991832&w=3840&q=75 3840w"
            style="color: transparent; opacity: 0; transition: opacity .5s linear; object-fit: cover;"
            width="400"
          />
        </li>
      </ul>
    </main>
    <footer
      class="footer"
    >
      <a
        href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        Powered by
         
        <span
          class="logo"
        >
          <img
            alt="Vercel Logo"
            data-nimg="1"
            decoding="async"
            height="16"
            loading="lazy"
            src="/vercel.svg"
            style="color: transparent;"
            width="72"
          />
        </span>
      </a>
    </footer>
  </div>
</div>
`;
